LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2017, Zend Technologies USA, Inc.
+Copyright (c) 2015-2018, Zend Technologies USA, Inc.
 All rights reserved.
```

Comment:
Sucks that we have to repeat it here, but that's how the BSD license is phrased. So we'll have it in two locations, essentially.